data/data/gcp/cluster/main.tf

```diff
@@ -100,6 +100,15 @@ resource "google_compute_image" "cluster" {
   guest_os_features {
     type = "UEFI_COMPATIBLE"
   }
+  guest_os_features {
+    type = "GVNIC"
+  }
+  guest_os_features {
+    type = "VIRTIO_SCSI_MULTIQUEUE"
+  }
+  guest_os_features {
+    type = "SEV_CAPABLE"
+  }
```

Comment:
I know we talked about this in slack, but it would be good to discuss in the PR. In most, if not all cases, GCP is not creating a new compute image--instead it uses a global image specified in the coreos stream. The changes here will only be applied when terraform is creating a new image. So we need either:

1. Additional  code in the tfvars that will set `gcp_preexisting_image` to false when we want to use a confidential VM
2. To update the global coreos image and rm this commit. If we want to go this route, can you update the commit message to mark it as WIP and describe the dependency.

Reply:
Agree, this should be part of the default coreos image upload.
I'll remove this commit